### PR TITLE
Add Systemd script for linux

### PR DESCRIPTION
**Pull request by kascote**
updated a bit the docs and added a script to launch clipper as a service on linux

### CODE REVIEW

**Comment by wincent** [comment]
Thanks a lot! Will merge this soon.

File: contrib/linux-systemd-service/clipper.service [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Clipper ~ Clipboard proxy
+
+[Service]
+ExecStart=/home/nelson/bin/clipper
```
Comment (wincent) [inline contrib/linux-systemd-service/clipper.service]: Can we change this to (and will it work with) `$HOME` or `~`?

Comment (kascote) [reply contrib/linux-systemd-service/clipper.service]: The ExecStart path must be absolute, don't support variable expansion or directory expansion (support it for the command arguments).  
Can be uses something like this too 

    ExecStart=/bin/sh -c '~/bin/clipper'

But that will left sh on memory too  

I updated the script to use a standard directory for the example.

**Comment by wincent** [comment]
Thanks again @kascote. This is now merged onto `master`.

**Pull request closed without merging**